test/cmdlineTests/asm_json/output

```diff
@@ -1582,5 +1582,10 @@ EVM assembly:
         }
       ]
     }
-  }
+  },
```

Comment:
Why is the removal of `"jumpType"` and addition of `"modifierDepth"` not reflected here?

Reply:
I'm now also wondering. Yesterday I changed exactly this part. The thing is that in our current tests we have it like this:
```json
        {
          "begin": 77,
          "end": 158,
          "name": "JUMP",
          "source": 0,
          "value": "[in]"
        }
```
So I saw yesterday, that the `jumpType` is stored in `value` - so I just changed that - without really checking the original code. The strange thing is that the original code reads like that there should be an `jumpType` node. Something seem to be wrong. I will look into this. Thx for that hint!

Reply:
For `modifierDepth` it is just needed to be able to import the asm json correctly. Without this information it is not possible to generate the original byte code.

Reply:
So that JSON the function creates is not the final form but later goes through some extra processing?

Reply:
Not sure what you mean. I think `modifierDepth` was just forgotten when `asm-json` was first implemented.

Reply:
Oh. From your comment I thought that `modifierDepth` is used internally but then not written out on purpose - so there must be some postprocessing step that removes it (and maybe does other stuff).

Reply:
I just needed to add `modifierDepth`. It was just missing in the original `asm-json` implementation. There is no post processing or any other extra processing involved. This information was just missing, and now this information will be exported correctly.

Reply:
Just to also write it here in this thread: decision from call: always store jump type in a `jumpType` node. @cameel thx for making us ware of that!